clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -483,8 +487,19 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
                         "Kernel event marker value missing");
         if (memref.marker.marker_type == TRACE_MARKER_TYPE_KERNEL_EVENT &&
             // Give up on back-to-back signals.
```

Comment:
Add a XXX or TODO comment about adding checks for back-to-back?

Reply:
Done.